docs/modules/operation/pages/deep-dive/notifications/concepts.adoc

```diff
@@ -94,3 +94,55 @@ This switch specifies the logic used when deciding whether to send a notice for
 If the switch is `off`, notices will never be sent to the target.
 If the switch is `on`, notices will always be sent to the target.
 If the switch is set to `auto`, the system deploys heuristics aimed at "doing the right thing."
+
+== Variable bindings filters
+
+Event parameters can additionally act as a filter method.
+This can be helpful for events that don't have an association to a specific node and the common filters only match against interfaces.
+By defining the `vbname` and `vbvalue`, the notification will only be sent with if the event parameter matches the given value.
+
+Example 1:
+
+In this example, notifications on a SNMP Link Down event would be suppressed for trunk ports.
```

Comment:
Wouldn't this be the opposite, only triggering events that do match `*Trunk*`?

Reply:
Yes. It is the positive match.

```suggestion
In this example, notifications would only be triggered on a SNMP Link Down events that contain the string `Trunk`.
```